test/cmd/process.sh

```diff
@@ -6,7 +6,7 @@ trap os::test::junit::reconcile_output EXIT
 (
   set +e
   oc delete all,templates --all
-  oc delete user someval
+  oc delete user someval someval=moreval someval=moreval2 someval=moreval3
```

Comment:
A test in that file creates these users but doesn't delete them thus you can't run that file twice in a row. Not related to the envfile feature at all.